src/components/AppRoutes/AppRoutes.tsx

```diff
@@ -0,0 +1,22 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { PeoplePage } from '../Pages/PeoplePage';
+import { HomePage } from '../Pages/HomePage';
+
+export const AppRoutes = () => {
+  return (
```

Comment:
if you don't have any logic just jsx, then please return jsx without using word `return`, in round brackets

check other components too